lib/config/config-ops.js

```diff
@@ -159,7 +159,7 @@ module.exports = {
                     if (isRule) {
                         dst[i] = e;
                     } else {
-                        dst[i] = deepmerge(target[i], e, combine, isRule);
+                        dst[i] = ConfigOps.merge(target[i], e, combine, isRule);
```

Comment:
I wish I could claim I did it for some smart reason, but I just did it without thinking. `this` would work equally well.